en/docs/install-and-setup/setup/sso/okta-as-an-external-idp-using-saml.md

```diff
@@ -1,18 +1,22 @@
-In the following document we will explain how to connect OKTA as a third party Identity provier to WSO2 API-Manager. Before we start first make sure you have all the pre-requisites mentioned below.
+# Using OKTA as an External IDP with SAML
+
+In the following document we will explain how to connect OKTA as a third party Identity provier to WSO2 API Manager.
```

Comment:
@ruthryi , can you please review this entire document and update it, because it is not structured properly. You can use the en/docs/install-and-setup/setup/sso/okta-as-an-external-idp-using-oidc.md document that I reviewed as an example.